src/test/java/com/library/steps/B28G31_148_StepDefs_BorrowedTheMost_ZU.java

```diff
@@ -0,0 +1,53 @@
+package com.library.steps;
+
+
+import com.library.pages.LoginPage;
+import com.library.utility.DB_Util;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+
+
+public class B28G31_148_StepDefs_BorrowedTheMost_ZU {
+
+    LoginPage loginPage = new LoginPage();
+
+
+    @Given("the {string} on the home page ZU")
+    public void the_on_the_home_page_ag(String userType) {
+
+        loginPage.login(userType);
+
+
+    }
```

Comment:
You dont have this step in your feature file. Why did you add it here. The user does not need to login to the application in your user story